source/components/Link.tsx

```diff
@@ -0,0 +1,26 @@
+/*
+	This module is used to extend the react-router-dom lib by overriding
+	the Link component to keep the current search parameters when navigating.
+*/
+import { Link as LinkReact, useSearchParams } from 'react-router-dom'
+
+const persistentSearchParams = ['lang']
+
+export const Link = ({ to, children, ...props }) => {
+	const [searchParams] = useSearchParams()
+
+	for (let key of searchParams.keys()) {
```

Comment:
@EmileRolley I have the impression that this new Link component breaks the back navigation after clicking on the ScoreBar to come back to the simulation. 


> http://localhost:8080/fin?details=t2.52a2.03l1.41s1.11d0.64n0.21%3Flang%3Dfr&lang=fr

It produces this URL, with a strange "lang" duplicate

Reply:
It's fixed by https://github.com/datagir/nosgestesclimat-site/pull/641/commits/7301ca787a9dd5ac3e00037d748545110a029106 :heavy_check_mark: